src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -619,7 +630,7 @@ public DataAccessRequest populateProgressReportFromJsonString(String json,
 
     String referenceId = UUID.randomUUID().toString();
     newDar.setReferenceId(referenceId);
-    newDar.setParentId(parentDar.getId().toString());
+    newDar.setParentId(parentDar.getId());
```

Comment:
@raejohanek , @rushtong - I think this is a point of confusion.  The DAR body takes the parentId, the endpoint takes the referenceId of the parent DAR.  When we convert parentId, we might want to be more explicit with this and call it parentReferenceId.  This should help disambiguate from the numeric `id` property we're using now.